messages/2.0.0.txt

```diff
@@ -0,0 +1,7 @@
+=> 2.0.0
+
+⚠️ To ensure that everything works properly after LSP package is updated, it's strongly recommended to restart
+Sublime Text once it finishes updating all packages. ⚠️
+
+# Breaking changes
+- We are transitioning LSP and LSP-* packages from Python 3.3 to Python 3.8.
```

Comment:
Would include info on how this affects users.

I guess the most apparent side effect is that ST will require an extra restart for things to start working properly. Maybe the original warning should be updated to reflect that since for this case it will behave differently than normally.

Reply:
I did expand the release message to include info about what the user can expect in this update
and what the user can do if problem arise during the update.

Reply:
[b8d867c](https://github.com/sublimelsp/LSP/pull/2418/commits/b8d867c48b4d69435a1e443b0614a0ef5288b063)